plugins/nodes/src/js/data/NetworkNodesClient.ts

```diff
@@ -0,0 +1,32 @@
+import { Observable } from "rxjs";
+import { request, RequestResponse } from "@dcos/http-service";
+import { map } from "rxjs/operators";
+
+export interface NetworkNode {
+  updated: string;
+  public_ips: string[];
+  private_ip: string;
+  hostname: string;
+}
+
+export type NetworkNodesResponse = NetworkNode[];
+
+export function fetchNetworkNodes(): Observable<RequestResponse<string[]>> {
+  return request("/net/v1/nodes").pipe(
+    map((reqResp: RequestResponse<any>) => {
+      if (reqResp.code >= 300) {
+        const respMessage =
+          reqResp.response && typeof reqResp.response === "object"
```

Comment:
In what situation `http-service` returns a plain string?

Reply:
I followed this example https://github.com/dcos/dcos-ui/blob/master/plugins/services/src/js/data/ServicePlansClient.ts